Library/Formula/pyenv.rb

```diff
@@ -1,8 +1,8 @@
 class Pyenv < Formula
   homepage "https://github.com/yyuu/pyenv"
   head "https://github.com/yyuu/pyenv.git"
-  url "https://github.com/yyuu/pyenv/archive/v20150404.tar.gz"
-  sha256 "b7a2b2ef247e9ad66fb34985239fb0f40930bb924f091901475ef4b0731d1152"
+  url "https://github.com/yyuu/pyenv/archive/v20150524.tar.gz"
+  sha1 "0aaa13e9ed24a070383f20f7eacf696625fb9430"
```

Comment:
Sorry about that. I knew the older versions used `sha1` and used it again by mistake. I'll update.